R/foreign.R

```diff
@@ -596,12 +596,13 @@ write.graph.dot <- function(graph, file, ...) {
 #' @family foreign
 #' @export
 #' @keywords graphs
-graph_from_graphdb <- function(url = NULL,
-                               prefix = "iso", type = "r001", nodes = NULL, pair = "A", which = 0,
-                               base = "http://cneurocvs.rmki.kfki.hu/graphdb/gzip",
-                               compressed = TRUE, directed = TRUE) {
+graph_from_graphdb <- function(
```

Comment:
is this reformatting due to Air? :smile_cat:

Reply:
Yes :D. Should I revert it?

Reply:
No! I was just curious.